test/unit/calculators/benefit_cap_calculator_configuration_test.rb

```diff
@@ -136,6 +137,49 @@ class BenefitCapCalculatorConfigurationTest < ActiveSupport::TestCase
           end
         end
       end
+
+      context "location of user" do
+        context "lives outside of London" do
+          setup do
+            GdsApi::Imminence.any_instance.stubs(:areas_for_postcode).returns(
+              OpenStruct.new(
+                code: 200,
+                results: [
+                  {
+                    slug: "birmingham-city-council",
+                    country_name: "England"
+                  }
+                ]
+              )
+            )
+          end
```

Comment:
I think you should probably use `imminence_has_areas_for_postcode` instead of stubbing `GdsApi::Imminence`. Unless you've decided against doing that, in which case you should mention it in the commit note.

You'll need to use something like:

```
require 'gds_api/test_helpers/imminence'

include GdsApi::TestHelpers::Imminence

imminence_has_areas_for_postcode("B1%201PW", [{slug: 'birmingham-city-council', country_name: 'England'}])
```